source/index.html.erb

```diff
@@ -6,7 +6,7 @@ description: Take and process online payments from your users - GOV.UK Pay is a
     <nav class="breadcrumbs breadcrumbs--inverse" aria-label="Breadcrumbs" data-click-events data-click-category="Header" data-click-action="Breadcrumb clicked">
         <ol itemscope itemtype="http://schema.org/BreadcrumbList">
             <li class="breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
-                <a href="https://www.gov.uk/service-toolkit#components" itemprop="item"><span itemprop="name">Components</span></a>
+                <a href="https://www.gov.uk/service-toolkit#gov-uk-services" itemprop="item"><span itemprop="name">GOV.UK Services</span></a>
```

Comment:
Is it definitely “Services” with a capital “S”? I note that on https://www.gov.uk/service-toolkit#gov-uk-services and https://www.notifications.service.gov.uk/ it has a lower-case “s”.

Reply:
My mistake, should be sentence case.